boatd-opencpn

```diff
@@ -27,11 +27,16 @@ def nmea_line(line):
     return '$' + line + '*' + checksum
 
 
-def degrees_to_nmea(input_degrees):
+def degrees_to_nmea(input_degrees,places=3):
     '''Return a nmea formatted bearing from decimal degrees'''
     degrees = math.trunc(input_degrees)
     minutes = (input_degrees - degrees) * 60
-    return '{}{:.4f}'.format(degrees, abs(minutes))
+    if places == 2:
+        return '{:02}{:07.4f}'.format(degrees, abs(minutes))
```

Comment:
this could be written as `'{:0{places}}{:07.4f}'.format(degrees, abs(minutes), places=places)` to avoid duplication